website/templates/website/documentation_page.html

```diff
@@ -1,6 +1,15 @@
 {% extends "website/base_template.html" %}
 {% load customfilters %}
 
+{% block extra_css %}
+<style type="text/css">
+	.section {
+    padding-top: 50px;
```

Comment:
@jchoude yes actually I have kept a fixed offset of position. But It varies for big headings and small headings. I will try to find any better solution till then lets keep this PR open?